### PR TITLE
Standardize env var file format + add VS code extension

### PR DESCRIPTION
**Pull request by snario**

### CODE REVIEW

File: packages/devtools/.env.development [hunk 1]
```diff
@@ -1,7 +1,6 @@
+GANACHE_CACHE_FOLDER = '../../.ganache-deployments'
```
Comment (kerzhner) [inline packages/devtools/.env.development:1]: Can no longer use `source` to load `.env` files when there are spaces around `= `

Comment (snario) [reply packages/devtools/.env.development:1]: Do we ever use `source`?

Comment (kerzhner) [reply packages/devtools/.env.development:1]: Good question. We have in the past. Not sure we do at this point? If others do not use source, I am ok losing that functionality if this PR gives us a benefit like syntax highlighting.

Comment (kerzhner) [reply packages/devtools/.env.development:1]: There is also an argument for readability (regardless of syntax highlighting) with this PR.

File: packages/nitro-protocol/.env [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+SINGLE_ASSET_PAYMENT_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+CONSENSUS_APP_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TRIVIAL_APP_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_FORCE_MOVE_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_NITRO_ADJUDICATOR_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_TOKEN_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_ETH_ASSET_HOLDER_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_TOKEN_ASSET_HOLDER_ADDRESS = ' 0x0000000000000000000000000000000000000000'
+TEST_ASSET_HOLDER_ADDRESS  = ' 0x0000000000000000000000000000000000000000'
+TEST_ASSET_HOLDER2_ADDRESS = ' 0x0000000000000000000000000000000000000000'
```
Comment (geoknee) [inline packages/nitro-protocol/.env:20]:
```suggestion
TEST_ASSET_HOLDER2_ADDRESS = '0x0000000000000000000000000000000000000000'
```

etc? Nitpicking, for sure: but this PR is about standardising the format.

**Pull request closed without merging**